Python/bytecodes.c

```diff
@@ -304,6 +304,7 @@ dummy_func(
         };
 
         inst(BINARY_SUBSCR, (unused/4, container, sub -- res)) {
+            #if ENABLE_SPECIALIZATION
```

Comment:
Style nit: 

Typically, we unindent `#if`/`#else`/`#endif` one level for easier readability. I find this a bit easier to parse:

```suggestion
        #if ENABLE_SPECIALIZATION
```

Reply:
I know that's traditional, but honestly I'm not too happy with that convention and I find it less readable that what Irit proposes here. Also, she tried outdenting it in an early regmachine branch and the generated code was indentded weirdly.